tests/integration-tests/src/advisory_checker.rs

```diff
@@ -0,0 +1,282 @@
+use super::run_command;
+use regex::Regex;
+use std::path::PathBuf;
+use tempfile::TempDir;
+
+pub const ADVISORY_CHECKER_PATH: &str = env!("CARGO_BIN_FILE_ADVISORY_CHECKER");
+
+const SPEC_TEMPLATE: &str = r#"Name: {name}
+Version: {version}
+Release: 1
+Summary: Test package
+License: MIT
+
+%description
+Test package
+"#;
+
+fn create_spec_file(dir: &TempDir, name: &str, version: &str) -> PathBuf {
+    let content = SPEC_TEMPLATE
+        .replace("{name}", name)
+        .replace("{version}", version);
+    let path = dir.path().join("test.spec");
+    std::fs::write(&path, content).unwrap();
+    path
+}
+
+fn create_advisory_file(dir: &TempDir, filename: &str, content: &str) -> PathBuf {
+    let path = dir.path().join(filename);
+    if let Some(parent) = path.parent() {
+        std::fs::create_dir_all(parent).unwrap();
+    }
+    std::fs::write(&path, content).unwrap();
+    path
+}
+
+#[test]
+#[ignore]
+fn test_missing_advisories_dir_succeeds() {
+    let temp_dir = TempDir::new().unwrap();
+    let spec_file = create_spec_file(&temp_dir, "testpkg", "1.0.0");
+    let nonexistent_dir = temp_dir.path().join("nonexistent");
+
+    let output = run_command(
+        ADVISORY_CHECKER_PATH,
+        [
+            "--spec-file",
+            spec_file.to_str().unwrap(),
+            "--advisories-dir",
+            nonexistent_dir.to_str().unwrap(),
+        ],
+        [],
+    );
+
+    assert!(output.status.success());
+    let stdout = String::from_utf8_lossy(&output.stdout);
+
+    let output_pattern = r"Advisories directory .* not found, skipping";
```

Comment:
I'm glad that this behavior is documented but it has me wondering if we need to consider how to protect against cases where we accidentally point the checker at the wrong directory. Should the script fail if the directory doesn't exist? That seems like maybe a choice that would reduce the chance of making a mistake.